README.md

```diff
@@ -97,28 +94,13 @@ This modality is an optional installation parameter and will default to the `Mix
 mode cannot be changed once the contract has been installed.
 The mode is passed in as a `u8` value to `nft_holder_mode` runtime argument.
 
-#### WhitelistMode
-
-The `WhitelistMode` dictates if the contract whitelist restricting access to the mint entrypoint can be updated. There are currently
-two options:
-
-1. `Unlocked`: The contract whitelist is unlocked and can be updated via the set variables endpoint.
-2. `Locked`: The contract whitelist is locked and cannot be updated further.
-
-This `WhitelistMode` is an optional installation parameter and will be set to unlocked if not passed. However, the whitelist mode itself
-cannot be changed once the contract has been installed. The mode is passed in as a `u8` value to `whitelist_mode` runtime argument.
-
-| WhitelistMode | u8  |
-| ------------- | --- |
-| Unlocked      | 0   |
-| Locked        | 1   |
-
 #### Minting
 
 The minting mode governs the behavior of contract when minting new tokens. The minting modality provides two options:
 
 1. `Installer`: This mode restricts the ability to mint new NFT tokens only to the installing account of the NFT contract.
 2. `Public`: This mode allows any account to mint NFT tokens.
+3. `Acl`: This mode allows whitelisted account/contract/contract from a package to mint NFT tokens.
```

Comment:
```suggestion
3. `Acl`: This mode allows whitelisted accounts, contracts or contracts from a package to mint NFT tokens.
```

Reply:
Is Acl an acronym? If so, it should be written as ACL.

Reply:
Renamed to ACL thanks